README.md

```diff
@@ -40,16 +39,19 @@ This command creates:
 - A Service of `type: LoadBalancer` that points to the Contour instances
 - Depending on your configuration, new cloud resources -- for example, ELBs in AWS
 
-See also [TLS support](docs/tls.md) for details on configuring TLS support. TLS is available in Contour version 0.3 and later.
+See also [TLS support](docs/tls.md) for details on configuring TLS support for the services behind Contour.
```

Comment:
We've supported TLS for basically forever so maybe we can clean this up, nobody will run Contour 0.2